src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -298,6 +297,36 @@ public static class StandardTrainersCatalog
         /// ]]></format>
         /// </example>
         public static SdcaMulticlassClassificationTrainer Sdca(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
+        string labelColumnName = DefaultColumnNames.Label,
+                    string featureColumnName = DefaultColumnNames.Features,
+                    string exampleWeightColumnName = null,
+                    float? l2Regularization = null,
+                    float? l1Threshold = null,
+                    int? maximumNumberOfIterations = null)
+        {
+            Contracts.CheckValue(catalog, nameof(catalog));
+            var env = CatalogUtils.GetEnvironment(catalog);
+            return new SdcaMulticlassClassificationTrainer(env, labelColumnName, featureColumnName, exampleWeightColumnName, null, l2Regularization, l1Threshold, maximumNumberOfIterations);
+        }
+
+        /// <summary>
+        /// Predict a target using a linear multiclass classification model trained with <see cref="SdcaNonCalibratedMulticlassClassificationTrainer"/>.
+        /// </summary>
+        /// <param name="catalog">The multiclass classification catalog trainer object.</param>
+        /// <param name="labelColumnName">The name of the label column.</param>
+        /// <param name="featureColumnName">The name of the feature column.</param>
+        /// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>
+        /// <param name="loss">Loss function to be minimized. Defaults to <see cref="LogLoss"/> if not specified.</param>
+        /// <param name="l2Regularization">The L2 <a href='tmpurl_regularization'>regularization</a> hyperparameter.</param>
+        /// <param name="l1Threshold">The L1 <a href='tmpurl_regularization'>regularization</a> hyperparameter. Higher values will tend to lead to more sparse model.</param>
+        /// <param name="maximumNumberOfIterations">The maximum number of passes to perform over the data.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[SDCA](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/StochasticDualCoordinateAscent.cs)]
```

Comment:
>StochasticDualCoordinateAscent [](start = 127, length = 30)

Should you add new sample file for non calibrated version?

Reply:
I don't want to touch too many those APIs as they are being renamed and moved.

---
In reply to: [266641719](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266641719) [](ancestors = 266641719)

Reply:
Then don't add example? I don't know how our documentation people track their work, but looks like it has everything it's needed, but in reality it's not.

---
In reply to: [266650832](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266650832) [](ancestors = 266650832,266641719)

Reply:
They have it now in Iteration 7.

---
In reply to: [266987401](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266987401) [](ancestors = 266987401,266650832,266641719)

Reply:
This PR fixes a big wrong thing I have been doing for years. I personally consider example is rather less important.

---
In reply to: [266987401](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266987401) [](ancestors = 266987401,266650832,266641719)